calico_kubernetes/calico_kubernetes.py

```diff
@@ -591,20 +614,37 @@ def _label_to_tag(self, label_key, label_value):
         tag = self._escape_chars(tag)
         return tag
 
+
+def config_logger():
+    logging._srcfile = None
+    log_dir = '/var/log/calico/kubernetes'
+    if not os.path.exists(log_dir):
+        os.mkdir(log_dir)
```

Comment:
use `os.makedirs()` here instead - this will only work if /var/log/calico already exists.